samza-core/src/main/java/org/apache/samza/clustermanager/ContainerAllocator.java

```diff
@@ -110,7 +110,7 @@
    */
   protected final ResourceRequestState resourceRequestState;
   /**
-   * Tracks the expiration of a request for resources.
+   * Tracks the default expiration of a request for resources.
```

Comment:
the default value is in ClusterManagerConfig: DEFAULT_CONTAINER_REQUEST_TIMEOUT_MS, this tracks the expiration-threshold for a resource request.